Library/Formula/redis-leveldb.rb

```diff
@@ -0,0 +1,21 @@
+class RedisLeveldb < Formula
+  desc "Redis-protocol compatible frontend to leveldb"
+  homepage "https://github.com/KDr2/redis-leveldb"
+  url "https://github.com/KDr2/redis-leveldb/archive/v1.4.tar.gz"
+  sha256 "b34365ca5b788c47b116ea8f86a7a409b765440361b6c21a46161a66f631797c"
+  head "https://github.com/KDr2/redis-leveldb.git"
+
+  depends_on "libev"
+  depends_on "gmp"
+  depends_on "leveldb"
+
+  def install
+    inreplace "src/Makefile", "../vendor/libleveldb.a", Formula["leveldb"].lib+"libleveldb.a"
+    system "make", "LDFLAGS=-lsnappy -lgmp -lev"
```

Comment:
The brewed `libleveldb.a` doesn't link with snappy statically. So it's needed to add -lsnappy.
To avoid inreplacing the makefile ugly, I set LDFLAGS here.